src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -112,7 +111,7 @@ class WorkspacePageWithSections extends React.Component {
                     >
                         <View style={[styles.w100, styles.flex1]}>
 
-                            {this.props.children(hasVBA, policyID, isUsingECard)}
+                            {this.props.children(isLoadingVBA, hasVBA, policyID, isUsingECard)}
```

Comment:
Since we changed up the order of arguments we must also check each usage of `WorkspacePageWithSections`. Spotted a few we missed:

![2022-05-25_10-25-17](https://user-images.githubusercontent.com/32969087/170360957-5e2d63bf-9a67-45cb-86f1-b85dd4dea9c9.png)